zcash_primitives/CHANGELOG.md

```diff
@@ -101,7 +101,7 @@ and this library adheres to Rust's notion of
   - `impl Sum<&Amount> for Option<Amount>`
 
 ### Changed
-- MSRV is now 1.51.0.
+- MSRV is now 1.56.1.
```

Comment:
We need to bump this in every downstream crate too. See f822d7aabcab01d6097c1b75ea38764266ca395b for an example of how the MSRV bump commit should look (and also it should close #498).